docs/pages/build/building-from-github.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Trigger builds from the Expo GitHub App
+description: Learn how to trigger builds on EAS for your app using the Expo GitHub App.
+---
+
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+This document outlines how to trigger builds on EAS for your app from a CI environment such as GitHub Actions, Travis CI, and more.
+
+Before building with EAS on CI, we need to install and configure `eas-cli`. Then, we can trigger new builds with the `eas build` command.
+
+## Prerequisites
+
+> While in preview, this feature is only available to EAS subscribers.
```

Comment:
```suggestion
> **warning** While in preview, this feature is only available to EAS subscribers.
```

Alternatively, since this feature is in preview, we can remove Prerequisites section and add this warning at the top of the page. In some cases, it might be easy for folks to skip the prerequisites and dive into the guide.